### ISSUE TITLE
The pure css example in the readme.mdown has an @include which should not be there

### ISSUE THREAD

**Issue by mariusGundersen** [mirisuzanne/compass-animation#12]
on line 98, it should read:

``` css
@keyframes my-animation {
  0%, 100% {
    background-color: blue;
  }
  50% {
    background-color: red;
  }
}

body {
  animation: my-animation 10s infinite;
}
```

**Comment by mirisuzanne** [comment]
Fixed by 6511a1dd131d283d66319f131d1fb6ab91b31ef5

**Comment by mirisuzanne** [comment]
Thanks!
